Training/Program.cs

```diff
@@ -1,7 +1,39 @@
-namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+//This code gives LCM and GCD of given multiple numbers
+Console.Write ("Mention the Count of numbers for GCD calculation: ");
+int nNum = int.Parse (Console.ReadLine ());
+List<int> numbers = GetNumbers (nNum);
+Console.WriteLine ($"The GCD of all input numbers --> {GCDforNnum (numbers)}");
+Console.WriteLine ($"The LCM of all input numbers --> {LCMforNnum (numbers)}");
+List<int> GetNumbers (int nNum) {
+   List<int> numbers = new ();
+   for (int a = 1; a <= nNum; a++) {
+      Console.Write ($"Enter number {a}: ");
+      int num = int.Parse (Console.ReadLine ());
+      numbers.Add (num);
+   }
+   return numbers;
+}
+int GCD (int num1, int num2) {
+   if (num2 == 0)
+      return num1;
+   else
+      return GCD (num2, num1 % num2);

```

Comment:
Ternary conditional operator (?:) can be used here to minimize the code:
_return (num == 0 ? num1 :  GCD (num2, num1 % num2))_;  
For more information, check out MSDN